src/main/java/appeng/tile/crafting/TileMolecularAssembler.java

```diff
@@ -82,7 +85,9 @@
 	private static final int[] SIDES = { 0, 1, 2, 3, 4, 5, 6, 7, 8, 9 };
 
 	private final InventoryCrafting craftingInv;
-	private final AppEngInternalInventory inv = new AppEngInternalInventory( this, 9 + 2 );
+	private final AppEngInternalInventory gridInv = new AppEngInternalInventory( this, 9 + 1, 1, new CraftingGridFilter() );
+	private final AppEngInternalInventory patternInv = new AppEngInternalInventory( this, 1, 1 );
+	private final IItemHandlerModifiable internalInv = new CombinedInvWrapper(gridInv, patternInv);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') '(' is not followed by whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') ')' is not preceded with whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)